common/src/main/java/com/wynntils/wc/utils/WynnItemUtils.java

```diff
@@ -238,22 +156,106 @@ public static List<ItemIdentificationContainer> fromProfile(ItemProfile item) {
             line.append(new TextComponent(" " + IdentificationProfile.getAsLongName(idName))
                     .withStyle(ChatFormatting.GRAY));
 
-            ItemIdentificationContainer id = new ItemIdentificationContainer(
-                    item,
-                    idProfile,
-                    type,
-                    idName,
-                    0,
-                    type == IdentificationModifier.Integer,
-                    0,
-                    -1,
-                    line,
-                    line,
-                    line,
-                    line);
+            ItemIdentificationContainer id =
+                    new ItemIdentificationContainer(item, idProfile, type, idName, 0, 0, -1, line, line, line, line);
             ids.add(id);
         }
 
         return ids;
     }
+
+    /**
+     * Create the colored percentage component for an item ID
+     *
+     * @param percentage the percent roll of the ID
+     * @return the styled percentage text component
+     */
+    public static MutableComponent getPercentageTextComponent(float percentage) {
+        Style color = Style.EMPTY
+                .withColor(
+                        ItemStatInfoFeature.colorLerp
```

Comment:
kind of odd to have a feature be referenced in what is a static utility class

Reply:
If it has the potential to be use elsewhere then it should be moved to a general config class later

Reply:
Yeah, our config implementation is pretty up in the air right now so this is mostly a temporary solution

Reply:
Yeah, it needs to be completely finalized sometime

Reply:
Unrelated to pr, but do you think we could code the logic part of the configs, so we can have setting holders (perhaps unchangeable now, or with a command only), making features able to use it, so when we add config guis, we already have all features using config fields?

Reply:
Yeah I think whatever the equivalent of `SettingsHolder` from legacy (I'm thinking of `Configurable` as an interface name) could just be extended by `Feature` to support config fields

Reply:
I can't fully think that would turn out to be, same feature problems we discussed, but after the feature "rework", it could work (though it absolutely can work now, it would just be a bit strange)

Reply:
Yeah, we need a time where we can fully complete discussion on the idea of features and configs